apps/martamap/marta_map.star

```diff
@@ -0,0 +1,806 @@
+Author: InTheDaylight14
+"""
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("http.star", "http")
+load("cache.star", "cache")
+load("encoding/json.star", "json")
+load("secret.star", "secret")
+
+TEXT_COLOR = "#aaaaaa"
+DEFAULT_ORIENTATION_BOOL = False  #Default to horizontal
+DEFAULT_ARRIVALS = True
+DEFAULT_STATION = "Five Points"
+DEFAULT_SCROLL = True
+DEAFULT_DIRECTION = None
+
+# FONT = "CG-pixel-3x5-mono"
+FONT = "CG-pixel-4x5-mono"
+
+def main(config):
+    MARTA_API_URL = secret.decrypt("AV6+xWcEqpLr4r5xWHL+ENipBzxVuOwqPhBwALkpo2ySIP8LvhyYYjsLoSq484C+X+Q91GmnTkRZBVPcITGvJJRJxmomJAwy4ejsRATXKIMmJHQy29u3IXDATDXbDuMXDx2wLeQXtfpuwWf5qHNh5VLrnE3d3ZJTfP6mS9xBxo+CwffQNt3YGa3pkQTpI5ikAKotOr95vcBPQzw22E3yY7iMqjR72wsE8s730m9pLnFolxbNmyMKZ6DNh+XFqNiNNer02eodGjhvLRR74xk8A1Q72Q==")
+    trains = get_trains(MARTA_API_URL)
+    arrivals = config.get("arrivals") or DEFAULT_ARRIVALS
+    orientation_bool = config.get("orientation") or DEFAULT_ORIENTATION_BOOL
+
+    if orientation_bool:
+        orientation = "vertical"
+    else:
+        orientation = "horizontal"
+
+    train_plots = []
+
+    #Invisible Marquee with long text to force all Marquees to pause ~10 sec...
+    train_plots.append(
+        render.Marquee(
+            width = 30,
+            child = render.Text("000000000000000000000000000000000000000000000000", color = "#000"),
+        ),
+    )
+
+    if orientation == "horizontal" and arrivals:
+        train_plots.append(render_arrivals(config))
+
+    #Add track plots and five points station overlay
+    train_plots.append(render_line(RED_LINE_POINTS, COLOR_MAP["RED"], orientation))
+    train_plots.append(render_line(GOLD_LINE_POINTS, COLOR_MAP["GOLD"], orientation))
+    train_plots.append(render_line(GREEN_LINE_POINTS, COLOR_MAP["GREEN"], orientation))
+    train_plots.append(render_line(BLUE_LINE_POINTS, COLOR_MAP["BLUE"], orientation))
+    train_plots.append(render_line(RED_LINE_FIVE_POINTS, COLOR_MAP["RED"], orientation))
+    train_plots.append(render_line(GOLD_LINE_FIVE_POINTS, COLOR_MAP["GOLD"], orientation))
+
+    #Add the train dots with real time data
+    for train in trains.values():
+        train_plots.append(render_train(train, orientation))
+
+    #Output 1 row and all the appended children centered on screen
+    return render.Root(
+        render.Row(
+            expanded = True,
+            main_align = "center",
+            cross_align = "center",
+            children = [render.Stack(
+                children = train_plots,
+            )],
+        ),
+    )
+
+def get_schema():
+    station_options = get_station_list_options()
+    direction_options = get_direction_list_options()
+
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Toggle(
+                id = "orientation",
+                name = "Vertical Orientation?",
+                desc = "Show map in vertical orientation?",
+                icon = "rectangle-vertical",
```

Comment:
Not sure what to use here instead. Check out https://github.com/tidbyt/community/blob/main/apps/icons.go.